pkg/generate/template.go

```diff
@@ -0,0 +1,67 @@
+package generate
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+var (
+	ErrCouldNotReadFile = errors.New("could not read file")
```

Comment:
suggestion: factor errors out into their own package and re-use them here and in `eval`

Reply:
Good idea, let me pencil out what that looks like.